apps/src/templates/DonorTeacherBanner.jsx

```diff
@@ -35,6 +37,7 @@ export default class DonorTeacherBanner extends Component {
         event: 'submit',
         data_string: $('input[name="nces-id"]').val(),
       });
+      analyticsReporter.sendEvent(EVENTS.AFE_HOMEPAGE_BANNER_SUBMIT);
```

Comment:
should the nces id also be logged, like above?

Reply:
I'm 90% convinced this component no longer has access to the nces id, which comes after the redirect on the next page. https://github.com/code-dot-org/code-dot-org/commit/d36bb6728f549c960a3b577d0945d31bac18b799